app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,8 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css" />
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'login', media: 'all', 'data-turbolinks-track': 'reload' %>
```

Comment:
instead of including  the `login.scss` in this file, you should import in the `application.scss`
https://www.w3schools.com/sass/sass_import.asp